Result.podspec

```diff
@@ -14,6 +14,6 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target = '2.0'
   s.tvos.deployment_target = '9.0'
   
-  s.swift_version = '4.0'
-  s.cocoapods_version = '>= 1.4.0'
+  s.swift_version = '4.1'
```

Comment:
This is for `SWIFT_LANGUAGE_VERSION` build setting, so 4.0 in Xcode 9.4 means Swift 4.1; it's not strange. But Xcode also accepts 4.1 as a valid value so this is okay.